packages/fast-breakpoint-tracker-react/package.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "@microsoft/fast-breakpoint-tracker-react",
+  "description": "React HOC for managing breakpoints",
```

Comment:
It's not really a HOC anymore - just a normal component

Reply:
Yeah - I'm updating this as well as adding a Readme per @janechu's comment